packages/react-core/src/components/Menu/MenuContainer.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Popper, PopperProps } from '../../helpers/Popper/Popper';
+
+export interface MenuContainerProps {
+  /** Menu to be rendered */
+  menu: React.ReactElement<any, string | React.JSXElementConstructor<any>>;
+  /** Reference to the menu */
+  menuRef: React.RefObject<any>;
+  /** Toggle to be rendered */
+  toggle: React.ReactNode;
+  /** Reference to the toggle */
+  toggleRef: React.RefObject<any>;
+  /** Flag to indicate if menu is opened.*/
+  isOpen: boolean;
+  /** Callback to change the open state of the menu.
+   * Triggered by clicking outside of the menu, or by pressing either tab or escape (or keys specified in onOpenChangeKeys). */
+  onOpenChange?: (isOpen: boolean) => void;
+  /** @beta Keys that trigger onOpenChange, defaults to tab and escape. */
+  onOpenChangeKeys?: string[];
```

Comment:
Just a nit: these descriptions could be updated to match the descriptions in Dropdown